src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="card-image"
+          class="card__image"
+        >
+      </div>
+      <h1 class="card__header header-position">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h1>
+      <h2 class="card__text text-position">
+        Код товара: 195434
+      </h2>
+      <div class="card__reviewes card__reviewes--position">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <h2 class="card__text ">Отзывов: 5</h2>
+      </div>
+      <div class="card__price card__price--position">
+        <span class="card__text card__text--sizeUp">Цена:</span>
```

Comment:
add all styles to card__text

```suggestion
        <span class="card__text">Цена:</span>
```